superset-frontend/plugins/plugin-chart-echarts/test/Gauge/transformProps.test.ts

```diff
@@ -342,3 +350,43 @@ describe('Echarts Gauge transformProps', () => {
     );
   });
 });
+
+describe('getIntervalBoundsAndColors', () => {
+  it('should generate correct interval bounds and colors', () => {
```

Comment:
nit/FYI: we're moving away from `describe` with nested `it`s towards simple `test` blocks. But we can convert these in a follow-up refactor pass.

Reply:
Good point, I wasn't aware of that. Will keep it in mind for the next PR :slightly_smiling_face: